components/CustomConnectWallet/CustomConnectWallet.tsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @next/next/no-img-element */
+
+import { ConnectButton } from "@rainbow-me/rainbowkit"
+
+const CustomConnectWallet = () => (
+  <ConnectButton.Custom>
+    {({
+      account,
+      chain,
+      openAccountModal,
+      openChainModal,
+      openConnectModal,
+      authenticationStatus,
+      mounted,
+    }) => {
+      // Note: If your app doesn't use authentication, you
+      // can remove all 'authenticationStatus' checks
```

Comment:
remove dev comments 🚮 

also, remove anything related to `authenticationStatus` as we are not using authentication 🚮